### PR TITLE
Created CITATION.cff

### LINKED ISSUE #1515: Jdaviz needs proper CITATION file
**Reporter:** pllim

This is a direct follow-up of https://github.com/spacetelescope/jdaviz/pull/1511 . A proper CITATION file is necessary to follow best practices as laid out in https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-citation-files .

To avoid controversy, first author should be a non-person, e.g., "Jdaviz developers" and the rest of contributors could be listed alphabetically by last name. 

[🐱](https://jira.stsci.edu/browse/JDAT-2620)

 --- 

 DISCLAIMER: This issue was autocreated by the Jdaviz Issue Creation Bot on behalf of the reporter. If any information is incorrect, please contact [Duy Nguyen](https://github.com/duytnguyendtn)

### PR DESCRIPTION
**Pull request by ojustino**
### Description

The lead author is "JDADF Developers," and others are listed in alphabetical order by surname. I inferred ORCIDs by searching ADS.

The list is almost everyone who has made a commit in the repository(*).  There was one user whose name I don't know. Are there others missing who should be listed? @camipacifici and @svolfman come to mind.

It would be helpful if people to looked at their entries to make sure their name appears as they'd like it, with the correct ORCID (if they have one).

Again, we would have to make sure to update the `date-released` and `version` lines with every new release.

Would fix #1515.

### Checklist for package maintainer(s)

This checklist is meant to remind the package maintainer(s) who will review this pull request of some common things to look for. This list is not exhaustive.

- [ ] Are two approvals required? Branch protection rule does not check for the second approval. If a second approval is not necessary, please apply the `trivial` label.
- [ ] Do the proposed changes actually accomplish desired goals? Also manually run the affected example notebooks, if necessary.
- [ ] Do the proposed changes follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
- [ ] Are tests added/updated as required? If so, do they follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
- [ ] Are docs added/updated as required? If so, do they follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
- [ ] Did the CI pass? If not, are the failures related?
- [ ] Is a change log needed? If yes, is it added to `CHANGES.rst`?
- [ ] Is a milestone set?
- [ ] After merge, any internal documentations need updating (e.g., JIRA, Innerspace)?

### CODE REVIEW

File: CITATION.cff [hunk 1]
```diff
@@ -0,0 +1,56 @@
+cff-version: 1.2.0
+message: "If you use this software, please cite it as below."
+authors:
+- family-names: "JDADF Developers"
+- family-names: "@brechmos"
+  given-names: "@brechmos"
```
Comment (pllim) [inline CITATION.cff:6]:
```suggestion
- family-names: "Jones"
  given-names: "Craig"
```

**Review by pllim — COMMENTED** [review]
Pretty complete first pass, thanks!

Might want to circulate this on Slack and have a period for comments.

File: CITATION.cff [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  orcid: "https://orcid.org/0000-0003-1509-9966"
+- family-names: "Jones"
+  given-names: "Craig"
+- family-names: "Lim"
+  given-names: "Pey Lian"
+  orcid: "https://orcid.org/0000-0003-0079-4114"
+- family-names: "Nguyen"
+  given-names: "Duy"
+- family-names: "O'Steen"
+  given-names: "Ricky"
```
Comment (rosteen) [inline CITATION.cff:37]:
```suggestion
  given-names: "Richard"
  orcid: "https://orcid.org/0000-0002-2432-8946"
```

File: CITATION.cff [hunk 3]
```diff
@@ -0,0 +1,58 @@
+cff-version: 1.2.0
+message: "If you use this software, please cite it as below."
+authors:
+- family-names: "JDADF Developers"
+- family-names: "Averbukh"
```
Comment (pllim) [inline CITATION.cff:5]: @javerbukh

File: CITATION.cff [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+  given-names: "Jesse"
+- family-names: "Bradley"
+  given-names: "Larry"
+  orcid: "https://orcid.org/0000-0002-7908-9284"
+- family-names: "Buikhuizen"
```
Comment (pllim) [inline CITATION.cff:10]: @mariobuikhuizen

File: CITATION.cff [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+  given-names: "Mario"
+- family-names: "Busko"
```
Comment (pllim) [inline CITATION.cff:12]: @ibusko

File: CITATION.cff [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+- family-names: "Earl"
+  given-names: "Nicholas"
+  orcid: "https://orcid.org/0000-0003-1714-7415"
+- family-names: "Fox"
+  given-names: "Ori"
+  orcid: "https://orcid.org/0000-0003-2238-1572"
+- family-names: "Geda"
+  given-names: "Robel"
+  orcid: "https://orcid.org/0000-0003-1509-9966"
+- family-names: "Jones"
```
Comment (pllim) [inline CITATION.cff:29]: @brechmos

File: CITATION.cff [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+- family-names: "Lim"
+  given-names: "Pey Lian"
+  orcid: "https://orcid.org/0000-0003-0079-4114"
+- family-names: "Nguyen"
+  given-names: "Duy"
+  orcid: "https://orcid.org/0000-0002-1534-336X"
+- family-names: "O'Steen"
+  given-names: "Richard"
+  orcid: "https://orcid.org/0000-0002-2432-8946"
+- family-names: "Ogaz"
```
Comment (pllim) [inline CITATION.cff:40]: @SaOgaz

**Review by pllim — COMMENTED** [review]
Thanks for putting this together, @ojustino !

I am pinging those listed here without ORCID in case they want to provide one.

File: CITATION.cff [hunk 8]
```diff
@@ -0,0 +1,58 @@
+cff-version: 1.2.0
+message: "If you use this software, please cite it as below."
+authors:
+- family-names: "JDADF Developers"
+- family-names: "Averbukh"
+  given-names: "Jesse"
```
Comment (javerbukh) [inline CITATION.cff:6]:
```suggestion
  given-names: "Jesse"
  orcid: "https://orcid.org/0000-0002-0041-0363"
```

**Comment by PatrickOgle** [comment]
My Orcid entry looks correct.

**Review by kecnry — APPROVED** [review]

**Review by javerbukh — APPROVED** [review]
I think this is ready to go even without tests 😉 Thanks for taking this on @ojustino!

**Pull request merged**